x-pack/plugins/uptime/server/lib/requests/search/list_of_monitors.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ChunkResult } from './monitor_summary_iterator';
+import { QueryContext } from './query_context';
+import { MonitorSummary } from '../../../../common/runtime_types/monitor';
+import { Ping } from '../../../../common/runtime_types/ping';
+import { summaryPingsToSummary } from './refine_potential_matches';
+import { createEsQuery } from '../../lib';
+
+export const getQueryFilter = (query?: string) => {
+  return query
+    ? {
+        minimum_should_match: 1,
+        should: [
+          {
+            multi_match: {
+              query: escape(query),
```

Comment:
We should avoid escape per: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape .

Why do we need to escape the plain text here by the way?